subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap.c

```diff
@@ -415,7 +415,10 @@ int nrf_cloud_coap_location_get(struct nrf_cloud_rest_location_request const *co
 	size_t len = sizeof(buffer);
 	int err;
 
-	err = coap_codec_ground_fix_req_encode(request->cell_info, request->wifi_info, buffer, &len,
+	err = coap_codec_ground_fix_req_encode(request->cell_info,
+					       request->wifi_info,
+					       request->config,
+					       buffer, &len,
```

Comment:
`coap_codec.c` changes are likely missing from the PR. CI also fails because of that now.

Reply:
Premature push.

Reply:
CoAP passing of config flags needs to be via a query string for now. Refactored all this and added a central function to generate the URI for both CoAP and REST.